content/200-concepts/100-components/01-prisma-schema/06-relations/420-referential-integrity.mdx

```diff
@@ -8,21 +8,53 @@ preview: true
 
 <TopBlock>
 
-Referential integrity means a set of constraints or triggers to handle the consistency of data of a [`@relation`](/reference/api-reference/prisma-schema-reference#relation). It can mean preventing a deletion of a record referred from other records, or it can be a cascading action defined in the field's [`referential actions`](/concepts/components/prisma-schema/relations/referential-actions). 
+To keep data consistent, it is necessary to have rules for what happens when related records are deleted or updated. For example, a `Post` model will need to define what happens if its author is deleted. **Referential integrity** refers to the set of constraints and triggers used to enforce this consistency.
 
-The referential integrity can either be handled in the database using foreign keys, or in the Prisma Client. There are pros and cons on both. In general, if the database supports foreign keys, it is usually the preferred choice and the default for SQL databases when using Prisma.
+In Prisma, relations between records are defined using the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attribute. The `onUpdate` and `onDelete` arguments of this attribute can be used to define which **referential actions** take place on updating or deleting the related record.
```

Comment:
Why is this relevant here? I think this could use some connection back, that we talk about this here because relations are what ref integrity is about. THis is not explicitly spelled out in the first paragraph before this one.